contracts/test/gateway/e2e-strategy-tests/Constants.sol

```diff
@@ -9,5 +9,5 @@ library Constants {
     address constant WBTC_ADDRESS = 0x03C7054BCB39f7b2e5B2c7AcB37583e32D70Cfa3;
     address constant TBTC_ADDRESS = 0xBBa2eF945D523C4e2608C9E1214C2Cc64D4fc2e2;
     address constant DUMMY_SENDER = 0x999999cf1046e68e36E1aA2E0E07105eDDD1f08E;
-    address constant DUMMY_RECEIVER = 0x0000000000000000000000000000000000000000;
+    address constant DUMMY_RECEIVER = 0x0000000000000000000000000000000000000001;
```

Comment:
Any reason to change it? 
I assume `teller` contract has some checks to not allow `0` address

Reply:
No its that you cannot transfer erc20s to the zero address. fails with:
 `revert: ERC20: transfer to the zero address`

Do the fork tests all work for you while using the zero address there?

Reply:
Hmm, yeah, the end-to-end tests are failing. We don’t run them in CI, so they were missed. We should probably add them to CI. 
Can you create an issue for it so we don't forget?

Reply:
removed change and did on separate pr: https://github.com/bob-collective/bob/pull/536